packages/apps/docs/src/docs/blogchain/2021/update-to-the-kadena-token-economic-model-2021-01-29.md

```diff
@@ -58,7 +58,7 @@ subsequent blocks are mined from. A significant tranche of the token economic
 model is the **platform share**. Examples of uses of the platform share include:
 
 - Developer and economic grants, as seen in our
-  [Technical Grant Program](/blogchain/2020/announcing-kadena-technical-grants.2020-11-25)
+  [Technical Grant Program](/blogchain/2020/announcing-kadena-technical-grants)
```

Comment:
the standard for blog items is that the date is at the end of the filename. but with a '-' instead of a '.'